packages/packages.mk.addon_template

```diff
@@ -0,0 +1,54 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2018-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="[package name]"
+PKG_VERSION="[package upstream version]"
+PKG_REV="100"
+PKG_SHA256="[sha256 hash of the source file, downloaded from PKG_URL]"
+PKG_ARCH="any"
+PKG_LICENSE="[license of the upstream project]"
+PKG_SITE="[url to the upstream project, e.g. http://exmaple.com/libexample"
+PKG_URL="[download url, e.g. https://github.com/example/libexample/archive/$PKG_VERSION.tar.gz]"
+# PKG_MAINTAINER="John Doe (email)" # if you want to be know as maintainer for a addon
```

Comment:
"John Doe" # Full name or forum/GitHub nickname, if you want to be identified as the addon maintainer

^ we should stop using emails, it results in text-length issues in the Kodi GUI and it's just a bad idea to have emails collected these days (privacy, GDPR, etc.)